### ISSUE TITLE
Active development?

### ISSUE THREAD

**Issue by Zyphrax** [paypal/paypal-here-sdk-ios-distribution#120]
We are considering implementing the PayPal Here SDK in our POS product.

A few things worry me though:
- the (closed source) framework seems to be build with libstdc++ which is deprecated, see https://github.com/paypal/paypal-here-sdk-ios-distribution/issues/114
- my app is 21MB at the moment, the framework seems to **add 55 MB!** (excluding bundle)
- to use the framework with Swift you have to use a bridging header, the project doesn't have a modulemap
- the last release was in October 2016
- quite a few unanswered issues, there doesn't seem to be a lot of activity

I just want to make sure that we aren't spending time and money to integrate the SDK if PayPal is not going to update and support it.

**Comment by ppmtscory** [comment]
@Zyphrax this SDK is very much still in the loop with things.  Our resources and priorities are currently being occupied with a revamp of this SDK.  We're hoping to have the new & improved SDK available a little bit later this year.  Unfortunately I don't have an exact timeline for that, but just know it's coming.  Hence, since this SDK does work 'as is' (and many folks are using it successfully), we've moved focus to the newer version unless there's a severe 'breaking' issue that comes up.

As for Swift, I also used the bridging header and am, to be honest, not familiar with module maps.  What would be the advantage to using that over bridging header or would it simply be developer convenience/preference?

**Comment by Zyphrax** [comment]
@ppmtscory thank you, a revamped SDK sounds great!

Is there a specific reason that you are developing it closed source?
Or not at least partially open source?

For more information on module maps:
http://blog.cocoapods.org/Pod-Authors-Guide-to-CocoaPods-Frameworks/

**Comment by djMax** [comment]
I don't work at PayPal anymore (I used to), but part of the reason is that especially for EMV, these things require external certification by other entities. So if it's "easy" to change the behavior, it would likely fail certification. In the end it's just code, so I get that there's no such thing as inviolable code...

**Comment by Zyphrax** [comment]
I've started working on integrating the SDK.

First I had to disable Bitcode on my project, which is unfortunate.

After that I worked on code handling device events of the bluetooth reader and after about an hour of testing I ran into this issue: https://github.com/paypal/paypal-here-sdk-ios-distribution/issues/22

Now it keeps crashing (as described in the issue). It might stop when I restart the reader. This sounds like a "severe 'breaking' issue" to me.
